app/controllers/projects_controller.rb

```diff
@@ -26,16 +26,18 @@ def toggle_archive
     Project.find(params[:id]).toggle_archived!
   end
 
-  def duplicate
-    original = Project.includes(stories: :estimates).find(params[:id])
-    duplicate = original.dup
-    duplicate.title = "Copy of #{original.title}"
-    duplicate.save
+  def clone
+    @original = Project.includes(:projects, stories: :estimates).find(params[:id])
+  end
 
-    original.stories.each { |x| duplicate.stories.create(x.dup.attributes) }
+  def do_clone
```

Comment:
@arielj one suggestions for this methods named: `new_clone` and `clone`.

Reply:
thanks! I updated the names of the actions